requirements/requirements.txt

```diff
@@ -37,14 +38,13 @@ django-taggit==0.22.2
 django==1.11.16
 djangorestframework-yaml==1.0.3
 djangorestframework==3.7.7
-docutils==0.14            # via botocore
+future==0.16.0            # via django-radius
```

Comment:
This library is _only_ necessary in python2, and so this makes me realize something.  I'll bet the `pip-compile` we're using is using python2, when really we now want to use python3.

Can you try this again:

https://github.com/ansible/awx/blame/devel/requirements/README.md#L8

...but with `python3 -m venv /buildit` ?